core/domain/services/assets/CoreAssetManager.php

```diff
@@ -202,12 +204,20 @@ private function loadCoreJs()
             )
         )->setRequiresTranslation();
 
+        $this->addJavascript(
+            CoreAssetManager::JS_HANDLE_EE_VALUE_OBJECTS,
+            $this->registry->getJsUrl($this->domain->assetNamespace(), 'valueObjects'),
+            array(
+                CoreAssetManager::JS_HANDLE_EE_MODEL
+            )
+        )->setRequiresTranslation();
+
         $this->addJavascript(
             CoreAssetManager::JS_HANDLE_EE_HOC_COMPONENTS,
             $this->registry->getJsUrl($this->domain->assetNamespace(), 'hocComponents'),
             array(
                 CoreAssetManager::JS_HANDLE_EE_DATA_STORES,
-                CoreAssetManager::JS_HANDLE_EE_HELPERS,
+                CoreAssetManager::JS_HANDLE_EE_VALUE_OBJECTS,
```

Comment:
correct and yes expected.  It means that we can start using `@eventespresso/model` in our components instead of importing directly (and should lead to smaller bundle size for the components bundle).